src/pages/account/settings/locales/en-US.ts

```diff
@@ -0,0 +1,61 @@
+export default {
+  'accountandsettings.menuMap.basic': 'Basic Settings',
+  'accountandsettings.menuMap.security': 'Security Settings',
+  'accountandsettings.menuMap.binding': 'Account Binding',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'accountandsettings.menuMap.binding' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)